shell/platform/linux/fl_keyboard_handler.cc

```diff
@@ -538,14 +520,16 @@ gboolean fl_keyboard_handler_handle_event(FlKeyboardHandler* self,
   g_ptr_array_add(self->pending_responds, pending);
   FlKeyboardHandlerUserData* user_data = fl_keyboard_handler_user_data_new(
       self, fl_keyboard_pending_event_get_sequence_id(pending));
-  DispatchToResponderLoopContext data{
-      .event = event,
-      .specified_logical_key = fl_keyboard_layout_get_logical_key(
-          self->derived_layout, fl_key_event_get_group(event),
-          fl_key_event_get_keycode(event)),
-      .user_data = user_data,
-  };
-  g_ptr_array_foreach(self->responder_list, dispatch_to_responder, &data);
+  uint64_t specified_logical_key = fl_keyboard_layout_get_logical_key(
+      self->derived_layout, fl_key_event_get_group(event),
+      fl_key_event_get_keycode(event));
+  for (guint i = 0; i < self->responder_list->len; i++) {
```

Comment:
Presumably `responder_list` will never be null here?

Reply:
Correct, it is created in the constructor.